views/index.ecr

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="/css/application.css">
   </head>
 
-  <body>
+  <body onload="document.forms['search'].elements['search'].select();">
```

Comment:
`autofocus` [put the caret at the beginning of the field](http://jsfiddle.net/4xab9b8y/2/), that is not very convenient also.